src/components/about/homePage.tsx

```diff
@@ -80,9 +81,12 @@ const HomePage = () => {
           >
             <div className="font-seoul p flex flex-col justify-center space-y-6 pt-8 pb-8 text-center">
               <p>Find us on the Highlander Link:</p>
-              <a href="https://highlanderlink.ucr.edu/organization/rbrains">
-                https://highlanderlink.ucr.edu/organization/rbrains
-              </a>
+              <Link
```

Comment:
Instead of link use a button from our ReusableButton.tsx. It's already imported:

<img width="371" alt="image" src="https://github.com/user-attachments/assets/dde0f5d8-3ac6-427b-b768-68fbbf80c1f2" />


and use it like this:

<img width="261" alt="image" src="https://github.com/user-attachments/assets/8aaced9b-23b4-481f-80a6-db97de8d0030" />

It takes in three parameters: Padding, textsize, and the border color. This way all our buttons are consistent throughout the site.